inc/ulib_result.h

```diff
@@ -39,10 +41,15 @@ MU_DEFINE_ENUM(
     AZ_ULIB_SECURITY_ERROR             = (AZ_ULIB_ERROR_FLAG | 3),    /**<Security error */
     AZ_ULIB_SYSTEM_ERROR               = (AZ_ULIB_ERROR_FLAG | 4),    /**<System error */
     AZ_ULIB_CANCELLED_ERROR            = (AZ_ULIB_ERROR_FLAG | 5),    /**<Cancelled error */
-    AZ_ULIB_BUSY_ERROR                 = (AZ_ULIB_ERROR_FLAG | 6)     /**<Busy error */
+    AZ_ULIB_BUSY_ERROR                 = (AZ_ULIB_ERROR_FLAG | 6),    /**<Busy error */
+    AZ_ULIB_PRECONDITION_ERROR         = (AZ_ULIB_ERROR_FLAG | 7),    /**<Provided etag didn't match the expected one */
```

Comment:
>Provided etag didn't match the expected one [](start = 74, length = 43)

this description doesn't seem to fit the wider possibilities where this would be used

Reply:
Sure. :)... I'll fix it.

---
In reply to: [338273812](https://github.com/Azure/azure-ulib-c/pull/40#discussion_r338273812) [](ancestors = 338273812)